bodhi/client/__init__.py

```diff
@@ -301,9 +303,19 @@ def query(url, **kwargs):
     Args:
         url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
                        True.
+        mine (Boolean): If the --mine flag was set
```

Comment:
done. We were not documenting any options for overrides query, so i also added the --user option here too.